assembl/static2/js/app/components/administration/survey/mediaForm.jsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Translate, I18n } from 'react-redux-i18n';
+import { FormGroup, Checkbox } from 'react-bootstrap';
+
+import {
+  toggleMedia,
+  updateMediaHtmlCode,
+  updateMediaDescriptionTop,
+  updateMediaDescriptionBottom,
+  updateMediaDescriptionSide,
+  updateMediaTitle
+} from '../../../actions/adminActions';
+import FormControlWithLabel from '../../common/formControlWithLabel';
+
+const MediaForm = ({
+  descriptionTop,
+  descriptionBottom,
+  descriptionSide,
+  hasMedia,
+  htmlCode,
+  selectedLocale,
+  title,
+  toggle,
+  updateDescriptionTop,
+  updateDescriptionBottom,
+  updateDescriptionSide,
+  updateTitle,
+  updateHtmlCode
+}) => {
+  const titlePh = `${I18n.t('administration.ph.title')} ${selectedLocale.toUpperCase()}`;
```

Comment:
For readability and DRY you can put selectedLocale.toUpperCase() in a variable as you use it several times